tests/integration_tests/datasets/api_tests.py

```diff
@@ -530,7 +530,7 @@ def test_create_dataset_item(self):
         self.login(username="admin")
         table_data = {
             "database": main_db.id,
-            "schema": "",
+            "schema": None,
```

Comment:
All the other tests have `""`, so I figured changing just one of these to `None` should be sufficient (no point in adding a full test for this)